bin/cli.js

```diff
@@ -14,9 +14,18 @@ if (argv._.length < 3) {
   process.exit(1);
 }
 
+// Parse from/to as either a Regex or a string
+const isRegexMatch = /.*\/([gimy]*)$/;
```

Comment:
I would say:

```
replace-in-file from to some/file.js,some/**/glob.js --isRegex
```

Where `from` then is the regular expression string.

Yes, agreed, the CLI and arrays is a bit hard. I figured at least to use the comma separated list for the files would be most useful.